tests/test_layout.py

```diff
@@ -12,8 +12,8 @@ class TestLayout:
     @pytest.mark.nondestructive
     @pytest.mark.xfail("'mozilla.com' in config.getvalue('base_url')",
```

Comment:
As fate would have it, the change has landed on production and this xfail is no longer needed.

Not sure what what @bobsilverberg was experiencing, adding the cmd line option seemed to wfm.